src/Func.cpp

```diff
@@ -1303,6 +1303,7 @@ Stage FuncRefVar::operator/=(Expr e) {
 }
 
 FuncRefVar::operator Expr() const {
+    //printf("func.name() == '%s'\n", func.name().c_str() );
```

Comment:
Never mind, I want to merge this now, so I'll just cut this line myself during the merge.